tests/terraform/checks/resource/ncp/example_LBTargetGroupHTTPS/main.tf

```diff
@@ -0,0 +1,54 @@
+resource "ncloud_lb_target_group" "pass" {
```

Comment:
I think this whole file doesn't belong to this check, right?

Reply:
Sorry. I quickly delete it